### PR TITLE
Some minor updates

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by cuducos** [comment]
Hi @patrickelectric — many thanks for this PR. I was waiting a but because with #162 and #142 probably some of these files would be deleted or changed. Once PR #183 is merged, can you review your changes to avoid conflicts and then I think we're good to merge?

**Comment by patrickelectric** [comment]
@cuducos, corrections done to be compatible with #183. Merged without problem with #142.
We are good to go.

File: src/get_family_names.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def write_raw_data(df):
         'id': 'congressperson_id',
         'Filiação': 'parents'})
 
-    if len(people_with_more_than_two_parents):
+    if people_with_more_than_two_parents:
```
Comment (cuducos) [inline src/get_family_names.py]: As this `people_with_more_than_two_parents` is a Pandas data frame, this change breaks the code:

```
Traceback (most recent call last):
  File "src/get_family_names.py", line 183, in <module>
    get_congresspeople_parents_names()
  File "src/get_family_names.py", line 179, in get_congresspeople_parents_names
    write_raw_data(df)
  File "src/get_family_names.py", line 142, in write_raw_data
    if people_with_more_than_two_parents:
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/generic.py", line 917, in __nonzero__
    .format(self.__class__.__name__))
ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

Would you mind rolling back or editing your code to work with Pandas data frame?

**Comment by patrickelectric** [comment]
@cuducos done.

**Comment by cuducos** [comment]
Hi @patrickelectric — testing your PR again I got an error when running your `src/clean_cnpj_info_dataset.py`:

```console
$ python src/clean_cnpj_info_dataset.py
Traceback (most recent call last):
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/indexes/base.py", line 2134, in get_loc
    return self._engine.get_loc(key)
  File "pandas/index.pyx", line 132, in pandas.index.IndexEngine.get_loc (pandas/index.c:4433)
  File "pandas/index.pyx", line 154, in pandas.index.IndexEngine.get_loc (pandas/index.c:4279)
  File "pandas/src/hashtable_class_helper.pxi", line 732, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:13742)
  File "pandas/src/hashtable_class_helper.pxi", line 740, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:13696)
KeyError: 'message'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "src/clean_cnpj_info_dataset.py", line 73, in <module>
    data[key] = data[key].astype('category')
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/frame.py", line 2059, in __getitem__
    return self._getitem_column(key)
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/frame.py", line 2066, in _getitem_column
    return self._get_item_cache(key)
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/generic.py", line 1386, in _get_item_cache
    values = self._data.get(item)
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/internals.py", line 3543, in get
    loc = self.items.get_loc(item)
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/indexes/base.py", line 2136, in get_loc
    return self._engine.get_loc(self._maybe_cast_indexer(key))
  File "pandas/index.pyx", line 132, in pandas.index.IndexEngine.get_loc (pandas/index.c:4433)
  File "pandas/index.pyx", line 154, in pandas.index.IndexEngine.get_loc (pandas/index.c:4279)
  File "pandas/src/hashtable_class_helper.pxi", line 732, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:13742)
  File "pandas/src/hashtable_class_helper.pxi", line 740, in pandas.hashtable.PyObjectHashTable.get_item (pandas/hashtable.c:13696)
KeyError: 'message'
``` 

TBH it doesn't look like it's your fault. Maybe the bug was already there, maybe the data being fetched has changed… IDK.

However I can't merge something if it's raising an error. Would you mind adding something like and test (a) if it works and (b) is data still consistent?

```python
for key in categories:
    try:
        data[key] = data[key].astype('category')
    except KeyError:
        pass
```

**Comment by jtemporal** [comment]
Hi @patrickelectric since #218 `clean_cnpj_info_dataset.py` was removed, would you mind fixing the conflicts so we can merge this?

I'm looking forward for that `try ... except` on the setup file ;)

**Comment by cuducos** [comment]
Closed by inactivity

**Pull request closed without merging**